psutil/_psutil_windows.c

```diff
@@ -1143,7 +1150,33 @@ psutil_proc_threads(PyObject *self, PyObject *args) {
                 PyErr_SetFromOSErrnoWithSyscall("GetThreadTimes");
                 goto error;
             }
-
+            // https://stackoverflow.com/a/41902967
+            // https://wiki.python.org/moin/WindowsCompilers
+            if (!get_thread_desc_set) {
+                get_thread_desc_set = 1;
+                windll = GetModuleHandle("Kernel32.dll");
+                if (windll) {
+                    get_thread_desc = (get_thread_desc_t) GetProcAddress(windll, "GetThreadDescription");
+                }
+                get_thread_desc = (get_thread_desc_t) GetProcAddress(
+                    GetModuleHandle("Kernel32.dll"), "GetThreadDescription"
+                );
```

Comment:
This stuff should be done differently. 
You should first define the prototype of `GetThreadDescription` in `psutil/arch/windows/ntext.h` then load `GetThreadDescription` and set it as a global name in `psutil/_psutil_common.c`.